tests/v2/validation/rbac/rbac.go

```diff
@@ -0,0 +1,82 @@
+package rbac
+
+import (
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	management "github.com/rancher/rancher/tests/framework/clients/rancher/generated/management/v3"
+	"github.com/rancher/rancher/tests/framework/extensions/namespaces"
+	"github.com/rancher/rancher/tests/framework/extensions/projects"
+	"github.com/rancher/rancher/tests/framework/extensions/users"
+	"github.com/rancher/rancher/tests/framework/pkg/namegenerator"
+)
+
+const (
+	defaultRandStringLength = 5
+)
+
+
+//Helper to create random names for namespaces & projects 
+func AppendRandomString(baseName string) string {
```

Comment:
We have this function somewhere else. I think it's provisioning there is no need to rewrite the function.